packages/story-editor/src/app/rightClickMenu/provider.js

```diff
@@ -613,20 +611,20 @@ function RightClickMenuProvider({ children }) {
     }
 
     const stylesReset = selectedElements
-      .map(
-        (element) =>
-          // only clear element styles for certain element types
-          CLEARABLE_ELEMENT_TYPES.includes(element.type) &&
-          clearElementStyles(element)
-      )
-      .some((styles) => Boolean(styles));
+      .map((element) => clearElementStyles(element))
+      .some(Boolean);
 
     // only show snackbar if any elements had styles reset
     if (stylesReset) {
       showSnackbar({
         actionLabel: __('Undo', 'web-stories'),
         dismissible: false,
-        message: __('Cleared style.', 'web-stories'),
+        message: _n(
+          'Cleared style.',
+          'Cleared styles.',
```

Comment:
💭 I think we don't need this to be plural.

Reply:
yea I think either makes sense

Reply:
In english it makes sense but we need both so it can be translated properly

Reply:
I think in english we'd still say 'cleared styles' if one singular style was cleared. However, this is based on the number of selected elements

So clearing styles from one element would show `Cleared style.` even if multiple styles were cleared.

I think this should always be `styles` in english. However in other languages it may be necessary to make that differentiation.

Reply:
oh, i thought you were asking if we needed the singular/plural structure for translation not asking what the proper english phrasing would be here. 

with your additional explanation I agree with you it should be styles.